plugin.go

```diff
@@ -114,15 +121,28 @@ func deleteCache() *exec.Cmd {
 	)
 }
 
-func remoteConfigCommand(config Remote) *exec.Cmd {
+func initCommand(config InitOptions) *exec.Cmd {
 	args := []string{
-		"remote",
-		"config",
-		fmt.Sprintf("-backend=%s", config.Backend),
+		"init",
+	}
+
+	if config.BackendConfig != "" {
+		args = append(args, fmt.Sprintf("-backend-config=%s", config.BackendConfig))
```

Comment:
The `-backend-config` option can be repeated multiple times when executing the command. Once we agree that this PR is a good way to go, I'll file another one that would allow for multiple ones.